src/models/connectionInfo.ts

```diff
@@ -137,25 +137,14 @@ export function getPicklistDetails(connCreds: IConnectionInfo): string {
  */
 export function getConnectionDisplayString(creds: IConnectionInfo): string {
 	// Update the connection text
-	let text: string;
-	if (creds.connectionString) {
-		// If a connection string is present, try to display the profile name
-		if ((<IConnectionProfile>creds).profileName) {
-			text = (<IConnectionProfile>creds).profileName;
```

Comment:
does the profile name not get displayed anymore?

Reply:
This is only the description part `[....]`- the original title does contain profile name as is...

![image](https://github.com/microsoft/vscode-mssql/assets/13396919/14a26c98-6cfa-4514-a739-4812de255029)